cogs/give_applicant.py

```diff
@@ -0,0 +1,122 @@
+"""Contains cog classes for giving the applicant role."""
```

Comment:
Update descriptions to state that the cog "makes a user into an applicant" rather than just "giving them the applicant role"